usmqe_tests/api/gluster/conftest.py

```diff
@@ -74,7 +86,8 @@ def valid_volume_name():
     return pytest.config.getini("usm_volume_name")
 
 
-@pytest.fixture(params=[None, "./,!@##$%^&*()__{}|:';/<*+>)("])
+# TODO(fbalak) as `./,!@##$%^&*()__{}|:';/<*+>)(` as parameter
```

Comment:
It is invalid input that is processed differently then the one that is already programmed.